### PR TITLE
Init celery application in app once + run call to blocking call in executor

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- [x] the celery application is initialized once at the beginning (will later need to check what happens if Rabbit/Redis are restarted) (but if scheduler is changed, then we might ditch this)
- [x] run call to celery inspect in executor

### CODE REVIEW

File: services/web/server/src/simcore_service_webserver/activity/celery_client.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        comp_settings.task_name,
+        broker=comp_settings.broker_url,
+        backend=comp_settings.result_backend,
+    )
+
+
+def setup(app: web.Application):
+    app[APP_CLIENT_CELERY_CLIENT_KEY] = celery_app = _create_celery_app(app)
+
+    assert celery_app  # nosec
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/activity/celery_client.py:35]: shouldn't you also close the Celery app?

See https://github.com/celery/celery/blob/master/celery/app/base.py#L344
They seem to implement also context manager

 could use ``app.cleanup_ctx`` for that

File: services/web/server/src/simcore_service_webserver/activity/celery_client.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import logging
+
+import tenacity
+from aiohttp import web
+from celery import Celery
+from tenacity import before_log, stop_after_attempt, wait_fixed
+
+from ..computation_config import ComputationSettings
+from ..computation_config import get_settings as get_computation_settings
+from .config import APP_CLIENT_CELERY_CLIENT_KEY
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/activity/celery_client.py:10]: MINOR: one thing I started doing is hiding this constant in the module  (i.e. ``__APP_CLIENT_CELERY_CLIENT_KEY__``  so it enforces the developer to use ``get_celery_client``.

**Review by pcrespov — APPROVED** [review]
flash SAN ⚡   !

**Review by GitHK — APPROVED** [review]
👍

File: services/web/server/src/simcore_service_webserver/activity/celery_client.py (hunk 2 above)
Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/activity/celery_client.py:11]: done

File: services/web/server/src/simcore_service_webserver/activity/celery_client.py (hunk 1 above)
Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/activity/celery_client.py:36]: very good point

Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/activity/celery_client.py]: done

**Pull request merged**